python-black.el

```diff
@@ -119,6 +119,16 @@ DISPLAY-ERRORS is non-nil, shows a buffer if the formatting fails."
   (and (= (point-min) beg)
        (= (point-max) end)))
 
+(defun python-black-org-mode-block (&optional display-errors)
```

Comment:
can you move this interactive function after the other interactive functions? that way internal helpers stay the bottom of the file